packages/blockchain-api/package.json

```diff
@@ -16,6 +16,7 @@
     "deploy": "./deploy.sh"
   },
   "dependencies": {
+    "@celo/contractkit": "0.1.5",
```

Comment:
I believe this should use 0.1.6 if we're keeping to the scheme we decided in #eng-typescript 
https://celo-org.slack.com/archives/CL6JFBALE/p1569937790004100

Reply:
Updated!

Reply:
Actually, sorry, looks like this isn't quite as decided as I thought it was. See continued discussion here:
https://celo-org.slack.com/archives/CL6JFBALE/p1570136686024300